hibike/devices/YogiBear/YogiBear.cpp

```diff
@@ -63,43 +64,19 @@ void loop() {
 
 uint32_t device_write(uint8_t param, uint8_t* data, size_t len) {
   switch (param) {
-    case ENABLE:
-      if (data[0] == 0) {
-        motorDisable();
-      } else {
-        motorEnable();
-      }
-      return sizeof(bool);
-      break;
-
-    case COMMAND_STATE: 
-      driveMode = data[0];
-
-      switch (driveMode) {
-        case MANUALDRIVE:
-          disablePID();
-          break;
-        case PID_VEL:
-          enableVel();
-          break;
-        case PID_POS:
-          enablePos();
-          break;
-        default:
-          driveMode = MANUALDRIVE;
-          disablePID();
-          break;
-      }
 
-      return sizeof(uint8_t);
-      break;
-
-    case DUTY_CYCLE: 
+    case DUTY_CYCLE:
+      motorEnable();
```

Comment:
it works alright if the duty cycle is 0. we want any input to begin the motors, in this case, after deadBand is merged in, the motor won't drive until past the dead band but we want them to be on.